Makefile

```diff
@@ -34,4 +34,7 @@ starknet-test: starknet-clean
 	@cd ./contracts/cairo/ && snforge test
 
 starknet-deploy: starknet-build
-	@./contracts/cairo/deploy.sh
\ No newline at end of file
+	@./contracts/cairo/deploy.shtest
+
```

Comment:
is there a .shtest file? or is this a placeholder value?

Reply:
what's a shtest?

Reply:
was a typo :-1: good catch